backend/main/test_utils/strategy_utils.go

```diff
@@ -201,31 +250,35 @@ func (otu *OverflowTestUtils) GenerateSingleVoteWithNFT(
 
 	vote := otu.CreateNFTVote(v, nftIds, contract)
 	balance := 100000000 * (accountNumber)
-	vote.Primary_account_balance = uint64(balance)
-	vote.Staking_balance = uint64(balance * 5)
-	vote.Block_height = uint64(0)
+	primary := uint64(balance)
+	staking := uint64(balance * 5)
+	blockHeight := uint64(0)
+
+	vote.PrimaryAccountBalance = &primary
+	vote.StakingBalance = &staking
+	vote.BlockHeight = &blockHeight
```

Comment:
planning to clean this up so there is less code duplication on these generate functions.